ogusa/tests/test_startyears.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import os
+import ogusa
+from ogusa import SS, TPI
+
+CUR_PATH = os.path.abspath(os.path.dirname(__file__))
+PUF_PATH = os.path.join(CUR_PATH, '../puf.csv')
+
+def test_diff_start_year():
+    from ogusa.scripts.execute import runner
+    # Monkey patch enforcement flag since small data won't pass checks
+    SS.ENFORCE_SOLUTION_CHECKS = False
+    TPI.ENFORCE_SOLUTION_CHECKS = False
+    output_base = "./OUTPUT"
+    input_dir = "./OUTPUT"
+    for year in [2013, 2017, 2026]:
+        print('year = ', year)
+        user_params = {'frisch': 0.41, 'debt_ratio_ss': 1.0,
+                       'start_year': year}
+        runner(output_base=output_base, baseline_dir=input_dir, test=True,
+               time_path=True, baseline=True, age_specific=False,
+               user_params=user_params, run_micro=True,
+               small_open=False, budget_balance=False, data=PUF_PATH)
```

Comment:
@jdebacker you may want to use the [`pytest.mark.parametrize`](https://docs.pytest.org/en/latest/parametrize.html) decorator here. There are a couple reasons to use this:
1. if year 2013 fails, years 2017 and 2026 will still run.
2. you can use the [xdist py.test plugin](https://docs.pytest.org/en/3.0.0/xdist.html) to run parametrized tests in parallel. Tax-Calculator uses this. There's some explanation on how to use it in the TC [TESTING.md](https://github.com/open-source-economics/Tax-Calculator/blob/master/TESTING.md).

Reply:
@hdoupe I'll look at doing this, (1) would be nice.  Re (2), I don't think that will work here as the tax function estimation done at each step already uses parallelization, so I think I'd run into the issue that subprocesses can't spawn their own subprocess.

Reply:
Ok sounds good. 

(2) Ah, I keep forgetting about that.

Reply:
@hdoupe See my edits to this test to incorporate `pytest.mark.parameterize`.  Appreciate the suggestion to do that.